py/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "lambdaworks_py"
+version = "0.1.0"
+edition = "2021"
```

Comment:
It may be a good idea to define these at the package level and inherit it in the crates.

Reply:
We added those lines to keep consistency with the other crates. However, we can change this in another PR.